resources/session02/homework/http_server.py

```diff
@@ -31,11 +35,20 @@ def parse_request(request):
     if method != "GET":
         raise NotImplementedError("We only accept GET")
     return uri
+    
 
 
 def resolve_uri(uri):
     """This method should return appropriate content and a mime type"""
-    return b"still broken", b"text/plain"
+    import mimetypes
+    mimetypes.guess_type('.html')
+    #uri = "/webroot/a_web_page.html"
+    try:
+        with open(uri, 'rb') as f:
+             return f       
+    except os.path.isfile(uri):
```

Comment:
An except clause needs to catch exceptions.  This should be a syntax error.  What you mean to write (I suspect) is something more like this:

```python
try:
    with open(uri, 'rb') as fh:
        return fh.read()
except IOError:
    # handle error condition
```